lib/screens/v2/widget_instance/dup_alert/serialize.ex

```diff
@@ -116,6 +131,40 @@ defmodule Screens.V2.WidgetInstance.DupAlert.Serialize do
     end
   end
 
+  @spec platform_closure_name(DupAlert.t()) :: String.t() | nil
+  defp platform_closure_name(t) do
+    stops_in_alert = Enum.map(t.alert.informed_entities, & &1.stop)
+
+    platform_names =
+      t
+      |> child_stops_for_affected_line()
+      |> Enum.filter(&(&1.id in stops_in_alert and &1.platform_name))
+      |> Enum.map(& &1.platform_name)
+
+    case platform_names do
+      [platform_name] ->
+        platform_name
+
+      [] ->
+        nil
+
+      multiple_platforms ->
+        if Enum.all?(multiple_platforms, &(&1 in @gl_westbound_platforms)) do
+          @gl_westbound_direction_name
+        else
```

Comment:
Think a Sentry warning could make sense here since our assumption is that we should never end up in this situation (aside from a malformed alert)